store/postgres/src/subgraph_store.rs

```diff
@@ -1054,13 +1054,18 @@ impl SubgraphStoreTrait for SubgraphStore {
 
     fn input_schema(&self, id: &DeploymentHash) -> Result<Arc<Schema>, StoreError> {
         let (store, site) = self.store(&id)?;
-        let info = store.subgraph_info(site.as_ref())?;
+        // TODO: ask if we need to use non-default version here, I think we don't because it's schema provided by user
+        let info = store.subgraph_info(site.as_ref(), &Default::default())?;
```

Comment:
- [ ] Do we need a version here? Is default (latest) enough?

I think we don't because it's schema provided by user